cmd/set.go

```diff
@@ -22,8 +22,8 @@ var setUnsetFlags = []cli.Flag{
 		"", "TORUS_ENVIRONMENT", true),
 	newSlicePlaceholder("service, s", "SERVICE", "Use this service.",
 		"default", "TORUS_SERVICE", true),
-	newSlicePlaceholder("user, u", "USER", "Use this user (identity).",
-		"*", "TORUS_USER", true),
+	newSlicePlaceholder("user, u", "USER", "Use this user.", "", "TORUS_USER", false),
+	newSlicePlaceholder("machine, m", "MACHINE", "Use this machine.", "", "TORUS_MACHINE", false),
```

Comment:
set no defaults to an empty identity, which is invalid :)
you need to bring back that asterisk